pyproject.toml

```diff
@@ -203,7 +203,7 @@ ldap = [
     "python-ldap",
 ]
 leveldb = [
-    "plyvel",
+    "plyvel; python_version < '3.12'",
```

Comment:
Plyvel 1.5.1 now able to build in python 3.12 as well as provide wheels for 3.12 🥳🥳 https://pypi.org/project/plyvel/1.5.1/#files

Reply:
Wheels for Linux on X86_64

Reply:
cool. Removed it